packages/opentelemetry-id-generator-aws-xray/test/AWSXrayIdGenerator.test.ts

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import { AWSXRayIdGenerator } from '../src';
+
+const idGenerator = new AWSXRayIdGenerator();
+
+describe('AwsXRayTraceId', () => {
+  let traceId1: string, traceId2: string;
+  let prevTime: number, currTime: number, nextTime: number;
+  const inValidTraceId = '00000000000000000000000000000000';
+  let sandbox: sinon.SinonSandbox;
```

Comment:
There should be no need to create a fresh sandbox. Just use the global one via `Sinon.stub()` and `Sinon.restore()`. same a few lines below.